frontend/src/features/user/transfer-ownership/TransferOwnershipModal.tsx

```diff
@@ -80,16 +100,14 @@ const useModalState = ({ onClose, reset, trigger }) => {
         },
       },
     )
-  }, [page, email, user])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Ooo is there a reason why we are disabling lint here for `refetch()` and `resetModal()` does it break anything 😅

Reply:
Oops no reason, forgot to remove it heh. Amended here [2b2d46f](https://github.com/opengovsg/FormSG/pull/6802/commits/2b2d46ff47c768738cd678bdad4f3e7ad3fe7e9e)